openedx/features/discounts/views.py

```diff
@@ -0,0 +1,70 @@
+"""
+The Discount API Views should return information about discounts that apply to the user and course.
+
+"""
+# -*- coding: utf-8 -*-
+
+from .applicability import can_recieve_discount, discount_percentage
+
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+
+from openedx.core.djangoapps.oauth_dispatch.jwt import create_jwt_for_user
+from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
+from openedx.core.lib.api.permissions import ApiKeyHeaderPermissionIsAuthenticated
+from openedx.core.lib.api.view_utils import DeveloperErrorViewMixin
+
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+
+class CourseUserDiscount(DeveloperErrorViewMixin, APIView):
+    """
+    **Use Cases**
+
+        Request discount information for a user and course
+
+    **Example Requests**
+
+        GET /api/discounts/v1/course/{course_key}
+
+    **Response Values**
+
+        Body consists of the following fields:
+
+
+    **Parameters:**
+
+        username:
+            The username of the specified user for whom the discount data
+            is being accessed.
+
+    **Returns**
+
+        * 200 on success with above fields.
```

Comment:
Do these responses actually get returned by the current code? If so, can we add tests for them?

Reply:
Removed.